rotkehlchen/db/upgrades/v36_v37.py

```diff
@@ -42,25 +45,46 @@ def _update_history_events_schema(write_cursor: 'DBCursor', conn: 'DBConnection'
 
         for entry in read_cursor:
             if entry[11] is None:
-                new_entries.append(entry[:11] + ('none',) + entry[12:])
+                new_entries.append(entry[:NEW_BASE_ENTRY_FIELDS_COUNT - 1] + ('none',))  # turn NULL values to text `none`  # noqa: E501
             else:
-                new_entries.append(entry)
+                new_entries.append(entry[:NEW_BASE_ENTRY_FIELDS_COUNT])  # Don't change NON-NULL values  # noqa: E501
 
-    write_cursor.executemany('INSERT INTO history_events_copy VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ? ,? ,? ,?)', new_entries)  # noqa: E501
+    write_cursor.executemany('INSERT INTO history_events_copy VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)', new_entries)  # noqa: E501
     write_cursor.execute('DROP TABLE history_events')
     write_cursor.execute('ALTER TABLE history_events_copy RENAME TO history_events')
     log.debug('Exit _update_history_events_schema')
 
 
+def _create_new_tables(write_cursor: 'DBCursor') -> None:
+    """
+    Creates evm_events_info table and moves there values previously stored in history_events.
+    """
+    log.debug('Enter _create_new_tables')
+    write_cursor.execute("""
+        CREATE TABLE IF NOT EXISTS evm_events_info(
+            identifier INTEGER PRIMARY KEY,
+            counterparty TEXT,
+            product TEXT,
+            address TEXT,
+            extra_data TEXT,
+            FOREIGN KEY(identifier) REFERENCES history_events(identifier) ON UPDATE CASCADE ON DELETE CASCADE
+        );
+    """)  # noqa: E501
```

Comment:
Ehm ... @nebolax @yabirgb where is the "and moves there values previously stored in history_events part" ?

Reply:
Then for the love of god add a comment here. And force a redecoding.